cmd/package-builder/package-builder.go

```diff
@@ -36,18 +40,28 @@ func runMake(args []string) error {
 		)
 		flOsqueryVersion = flagset.String(
 			"osquery_version",
-			env.String("OSQUERY_VERSION", ""),
+			env.String("OSQUERY_VERSION", "stable"),
 			"the osquery version to include in the resultant packages",
 		)
+		flLauncherVersion = flagset.String(
+			"launcher_version",
+			env.String("LAUNCHER_VERSION", "stable"),
+			"the launcher version to include in the resultant packages",
+		)
+		flExtensionVersion = flagset.String(
+			"extension_version",
+			env.String("EXTENSION_VERSION", "stable"),
+			"the extension version to include in the resultant packages",
+		)
 		flEnrollSecret = flagset.String(
 			"enroll_secret",
 			env.String("ENROLL_SECRET", ""),
 			"the string to be used as the server enrollment secret",
 		)
-		flMacPackageSigningKey = flagset.String(
+		flSigningKey = flagset.String(
 			"mac_package_signing_key",
-			env.String("MAC_PACKAGE_SIGNING_KEY", ""),
-			"the name of the key that should be used to sign mac packages",
+			env.String("SIGNING_KEY", ""),
+			"the name of the key that should be used to packages. Will be platform specific",
```

Comment:
i think you mean `the key that should be used to sign packages`, since you're not dynamically fetching from env based on a name

Reply:
Maybe. For OSX you do. I don't know the other platforms yet.